src/model.cpp

```diff
@@ -320,6 +320,19 @@ void Model::setParameters(const std::vector<realtype> &p) {
     unscaleParameters(originalParameters, pscale, unscaledParameters);
 }
 
+void Model::setParameters(const std::map<std::string, realtype> &p,
```

Comment:
`setParametersById`? Also implement `setParametersByName`?

Reply:
Not using it. But maybe good to add for symmetry...